extractAbi.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+rm -rf ./release/abi
+mkdir -p ./release/abi
+for full_filename in build/contracts/*.json; do
```

Comment:
the filename is x.json.abi.json,better remove the first json
the script creates abi for all the contracts, including internal files, worth to place them in separate folders